scripts/content.js

```diff
@@ -67,27 +71,38 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
 
                   var LaborScore=document.createElement('span');
                   LaborScore.classList.add('score');
+
+                  if(resp.result[index].LaborScore){
                   LaborScore.innerText=resp.result[index].LaborScore;
                   LaborScore.style["background-color"] =resp.colourMap[resp.result[index].LaborScore];
-
-
+                }
+                else{
+                  LaborScore.innerText="TBD";
+                  LaborScore.style["background-color"] =resp.colourMap[resp.result[index].LaborScore];
+                }
                   var LaborScoret=document.createElement('span');
                   LaborScoret.classList.add('scoret');
-                  LaborScoret.innerText=" LaborScore "
+                  LaborScoret.innerText=" Labor Rate "
```

Comment:
Apologies as I was unclear before, but this should be "Labor rating". To clarify the titles on the pop up should be:

### Overall Ratings
#### Labor Rating
#### Transparency Rating
#### Environment Rating

Reply:
Done :+1: